inception-project/inception
# Feature/2276 more spring auto configuration

**What's in the PR**
* Switch components and test over to auto-config

**How to test manually**
* Build with tests
* Check if the application boots
* No particular testing procedure

**Automatic testing**

**Documentation**
